dwallet/src/eth_state.rs

```diff
@@ -51,8 +51,6 @@ pub struct EthState {
     #[serde(default)]
     pub finalized_header: Header,
     #[serde(default)]
-    rpc: String,
```

Comment:
why was it removed?

Reply:
Instead of instantiating a new `NimbusRpc` every time it is needed, the RPC is now being held inside the `EthLightClient` and is injected whenever needed.